diff --git a/hal/spi/spi_drv_nrf52.h b/hal/spi/spi_drv_nrf52.h
index 33d31b7..9db3972 100644
--- a/hal/spi/spi_drv_nrf52.h
+++ b/hal/spi/spi_drv_nrf52.h
@@ -24,10 +24,10 @@
 
 /* SPI Pin Configuration (P0.x) */
 #if 1
-#define SPI_CS_PIN 13
-#define SPI_MOSI_PIN 4
-#define SPI_MISO_PIN 5
-#define SPI_SCLK_PIN 30
+#define SPI_CS_PIN 5
+#define SPI_MOSI_PIN 3
+#define SPI_MISO_PIN 4
+#define SPI_SCLK_PIN 2
 #endif
 #if 0
 #define SPI_SCLK_PIN 5
